requirements.txt

```diff
@@ -42,6 +42,7 @@ pytest==5.4.3
 pytest-mock==3.1.0
 pytz==2020.1
 PyYAML==5.1.2
+random-user-agent==1.0.1
```

Comment:
You updated requirements.txt, but we're also using the Pipfile. If possible it would be great if you can:
```
pipenv shell
pipenv install random-user-agent
```
And commit changes to the Pipfile and Pipfile.lock.

Reply:
Done in latest commit